### DISCUSSION TITLE
Launch dicom folder and Weasis from terminal Mac

### DISCUSSION THREAD

**Discussion by alfr3d1n0** [nroduit/Weasis#463] [Q&A]
Hi everyone, I've been launching Weasis from the Mac terminal for hours with the script "weasis-linux.sh" the folder on desktop.

./Viewer-linux.sh '$dicom:get -l "/users/alfredo/desktop/dicom"'

but without success.

In the same folder where the file "viewer-linux.sh" is present there is weasis.app.

How can I solve it?
Thanks

**Comment by nroduit** [comment]
The portable distribution was deprecated and completely removed in version 4 because it's too much effort to maintain on all systems and architectures. On Mac, the portable version was abandoned a long time ago because of the obscure security of applications (sandboxing, trusting...) that have to be constantly adjusted without clear rules.

**Reply by alfr3d1n0** [reply]
> The portable distribution was deprecated and completely removed in version 4 because it's too much effort to maintain on all systems and architectures. On Mac, the portable version was abandoned a long time ago because of the obscure security of applications (sandboxing, trusting...) that have to be constantly adjusted without clear rules.

While the iso file created by weasis for DVD still remains a reliable and working solution to date?

**Comment by nroduit** [comment]
Yes but only for Windows
